src/handler/s3Event.ts

```diff
@@ -11,15 +12,15 @@ import { createLogger, Logger } from '../util/logger';
  */
 export const handler = async (
   event: S3Event,
-  context: Context,
 ): Promise<Record<string, unknown>> => {
-  const bucketName = event.Records[0].s3.bucket.name;
-  const logger: Logger = createLogger(null, context);
+  const record: S3EventRecord = event.Records[0];
```

Comment:
Would it not be better doing a forEach on event.Records in the eventuality that there is more than 1 index in the array